webpack.config.js

```diff
@@ -11,5 +11,6 @@ module.exports = {
             exclude: /node_modules/,
             use: 'babel-loader'
         }]
-    }
+    },
+    mode: 'development'
```

Comment:
Hi @vrajeshkanna, thanks for taking out time to send in this PR. Everything looks good except this `development` flag here. 

I'm not sure if webpack build will work properly for `production` mode now (Check package.json scripts).

It'd best if we can specify this flag in the scripts instead, which will allow us to use single webpack config for development and production.

Reply:
Got it. I'll remove it from the config.js to let users add them to scripts at their will.